thewalrus/tests/test_samples.py

```diff
@@ -415,17 +433,19 @@ def test_thermal_state_torontonian(self, sample_func):
         probs[1] = 1 - probs[0]
         assert np.all(np.abs(rel_freq - probs) < rel_tol / np.sqrt(n_samples))
 
-    def test_torontonian_sample_graph(self):
+    @pytest.mark.parametrize("parallel", [True, False])
+    def test_torontonian_sample_graph(self, parallel):
         """Test torontonian sampling from a graph"""
-        # Pool does not play nicely with pytest when all the test are run together
         A = np.array([[0, 3.0 + 4j], [3.0 + 4j, 0]])
         n_samples = 1000
         mean_n = 0.5
-        pool = False
-        samples = torontonian_sample_graph(A, mean_n, samples=n_samples, pool=pool)
+        samples = torontonian_sample_graph(A, mean_n, samples=n_samples, parallel=parallel)
         assert np.all(samples[:, 0] == samples[:, 1])
 
 
+# def test_max_photons
+
+
```

Comment:
Should this be deleted?

Reply:
Yes! Sorry about that. Started writing a test and forgot about it. It's gone now!